pkg/operator/controller/certificate-publisher/controller.go

```diff
@@ -105,7 +105,7 @@ func New(mgr manager.Manager, operatorNamespace, operandNamespace string) (runti
 
 // secretToIngressController maps a secret to a slice of reconcile requests,
 // one request per ingresscontroller that references the secret.
-func (r *reconciler) secretToIngressController(o client.Object) []reconcile.Request {
+func (r *reconciler) secretToIngressController(ctx context.Context, o client.Object) []reconcile.Request {
```

Comment:
Since now the context is available in the enqueue function, it can be used in the cache calls down the code, instead of `context.Bakcground()`.

Reply:
Ah yes I missed this one, thanks.